src/index.ts

```diff
@@ -41,6 +51,51 @@ export interface KeyStore {
     clear(): Promise<void>;
 }
 
+export interface HDWKeyStore {
+    getSeedHashes(): Promise<SeedHash[]>;
+    importSeed(params: {
+        secret: SecretSeedStorage;
+        passphrase?: string;
+        meta?: string;
+    }): Promise<SeedHash>;
+    importRawSeed(params: {
+        seed: Seed;
+        passphrase?: string;
+        meta?: string;
+    }): Promise<SeedHash>;
+    exportSeed(params: {
+        seedHash: SeedHash;
+        passphrase: string;
+    }): Promise<SecretSeedStorage>;
+    exportRawSeed(params: {
+        seedHash: SeedHash;
+        passphrase: string;
+    }): Promise<Seed>;
+    createSeed(params: { passphrase?: string }): Promise<SeedHash>;
```

Comment:
I think we need `meta` field here.

```suggestion
    createSeed(params: { passphrase?: string, meta?: string }): Promise<SeedHash>;
```

Reply:
Fixed.